.travis.yml

```diff
@@ -16,7 +16,16 @@ env:
 script: test/k8s-local-cluster-test/run-test -v $KUBE_VERSION
 
 matrix:
-  include: 
+  include:
+    - stage: Test
+      language: go
+      go:
+        - 1.13
+      before_script:
+        - go get -t -v ./...
```

Comment:
I don't think you need this since we're using go modules

Reply:
Just set the `GO111MODULE: "on"` environment variable and, yes, you don't need the above.